src/ansys/acp/core/mechanical_integration_helpers.py

```diff
@@ -0,0 +1,162 @@
+# Copyright (C) 2022 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""TODO."""
+
+import pathlib
+import shutil
+import textwrap
+import typing
+
+if typing.TYPE_CHECKING:
+    import ansys.mechanical.core as pymechanical
+
+from ._typing_helper import PATH
+
+__all__ = [
+    "export_mesh_for_acp",
+    "import_acp_composite_definitions",
+    "import_acp_solid_model",
+]
+
+
+def export_mesh_for_acp(*, mechanical: "pymechanical.Mechanical", path: PATH) -> None:
+    """Export the mesh from PyMechanical for use in PyACP.
+
+    Parameters
+    ----------
+    mechanical :
+        The PyMechanical instance. This must be a remote instance.
+    path :
+        The path to save the mesh to. The extension must be '.h5'.
+    """
+    path = pathlib.Path(path)
+
+    if path.suffix != ".h5":
+        raise ValueError(f"The output path extension must be '.h5', not '{path.suffix}'.")
+    output_path_str = str(path)
+    mechanical.run_python_script(
+        textwrap.dedent(
+            f"""\
+            geometry_type = Ansys.Mechanical.DataModel.Enums.GeometryType.Sheet
+            unit = Ansys.Mechanical.DataModel.Enums.WBUnitSystemType.ConsistentMKS
+            dsid = 0
+
+            Model.InternalObject.WriteHDF5TransferFile(geometry_type, {output_path_str!r}, unit, dsid)
+            """
+        )
+    )
+
+
+def import_acp_solid_model(*, mechanical: "pymechanical.Mechanical", cdb_path: PATH) -> None:
+    """Import an ACP solid model into Mechanical.
+
+    Import a solid model in CDB format into Mechanical.
```

Comment:
We could comment that this imports the mesh and some of the NS but not the layup.